### ISSUE TITLE
Should we allow localized content?

### ISSUE THREAD

**Issue by saranshkataria** [fbdevelopercircles/FbDevcCommunityContent#3]
Though it is a great feature to have, but this list is going to be a huge one, and having localized content included in it would increase this list to a very long one as well. My opinion is maybe have a separate list for localized content and not have it in here.

**Comment by alicerp** [comment]
Would it make sense to create multiple branches in the repo in this case?

**Comment by aboullaite** [comment]
@alicerp I prefer having folders, for each specific language and keep everything in the master branch

**Comment by saranshkataria** [comment]
I agree with @aboullaite, though I think a file name with the language name in the name of the markdown file instead of having another folder would be sufficient.

**Comment by AJV009** [comment]
So am I allowed to add (PR) our developer circle member blogs and articles links to this file?

**Comment by Bello2609** [comment]
@aboullaite it will be nice if each languages have their own folder for easy navigation
